docs/static/include/pluginbody.asciidoc

```diff
@@ -779,6 +779,36 @@ will be discussed further in the testing section of this document.
 Another kind of external dependency is on jar files.  This will be described
 in the "Add a `gemspec` file" section.
 
+===== Dependencies on deprecated features
+
+Sometimes a plugin relies on a feature from another plugin. And sometimes those
+features get deprecated. {ls} provides an
+https://github.com/logstash-plugins/logstash-mixin-deprecation_logger_support[adapter]
+to make handling those situations easier. You can add this mixin as a dependency
+for any plugin that uses the deprecation logger so that the plugin continues to
+work with older versions of {ls}.
+
+*TO DO:* Add more intro text here about when/how that would happen.
+A use case or real world example would be awesome. 
+
+*TO DO:*  How much and where should we talk about the deprecation logger?: 
+
+* More info here? 
+* Somewhere else and link to it? 
+* Is the output of the deprecation logger so obvious and straightforward that we
+don't need to talk about it? 
+
+*TO DO:* Find out how/where somebody developing a plugin would know that a feature they
+need is deprecated.
+
+*TO DO:*  Is there any kind of log that would let a developer of an existing
+plugin know that something they rely on has been deprecated?  
```

Comment:
If the plugin is run on LogStash before `7.6` he has to check `logstash.log` in log directory, and inside that file search for `DEPRECATED` tagged lines.
On LogStash `7.6` and above there is a separated file, named `logstash-deprecation.log` always in `log` directory, that contains all the deprecation notices.